contents/docs/experiments/running-experiments-without-feature-flags.mdx

```diff
@@ -0,0 +1,238 @@
+---
+title: "Advanced: How to run experiments without feature flags"
+---
+
+You may want to run experiments without using PostHog's feature flags. There are 2 reasons why you may want to do this:
+
+1. You're using a different library for feature flags rather than PostHog's, and you want to run your experiment using this library.
+2. Feature flag support is not yet available in the PostHog SDK that you are using.
+
+This doc walks you through how to set up an experiment without using PostHog's feature flags.
+
+## Step 1: Create your experiment in PostHog
+
+Create a new experiment in the [experiments tab](https://app.posthog.com/experiments) and follow the same steps as you would when [creating a regular experiment](/docs/experiments/manual#how-to-create-an-experiment-in-posthog).
+
+An important callout is that you'll still need to name your feature flag key. Although you won't be using feature flags directly in your experiment, you'll still need to use this key when submitting events so that PostHog can calculate your experiment results.
+
+## Step 2 (optional): Call the `decide` endpoint
+
+If feature flag support is not yet available in your PostHog SDK, but you still want to leverage PostHog's feature flag infrastructure, a workaround is to use the PostHog [API](/docs/api) to call the [`decide`](/docs/api/post-only-endpoints#feature-flags) endpoint. This enables you to calculate the feature flag value for a given user.
+
+This step is not necessary if you are using your own feature flag library.
+
+## Step 3: Add the feature flag property to your events 
+
+In order to attribute events to their correct experiment variants, you'll need include a `$feature/experiment_feature_flag_key: variant-name` property when submitting your events. You only need to do this for events that you'd like to track in your experiment.
+
+This is similar to the [same step](/docs/experiments/manual#step-2-server-side-only-add-the-feature-flag-property-to-your-events) when setting up regular experiments. However, the key difference is that now you'll need to include this property when submitting events from all SDKs, and not just server-side SDKs.
+
+<MultiLanguage>
+
+```javascript
+posthog.capture('event_name_of_your_goal_metric', {
+    "$feature/experiment-feature-flag-key": "variant-name"
+})
+```
+
+```react
```

Comment:
Do we need a react example if it is exactly the same as JavaScript? I think not.

Reply:
Perhaps not, but I think it's nice to include. Makes it require less thinking